WTWSMS/history/titles/d_deheubarth.txt

```diff
@@ -0,0 +1,166 @@
+283.1.1={
+	holder = 159428 #Einudd ap Gwrddwfn
+}
+300.1.1={
+	holder = 159429 #Eudaf Hen
+}
+360.1.1={
+	holder = 159807 #Macsen Wledig
+}
+380.1.1={
+	holder = 159808 #Annwn Dynod
+}
+428.1.1={
+	holder = 159813 #Tudwal ap Annwn
+	set_global_flag = deheubarth_high_tribal
+}
+431.1.1={
+	holder = 159821 # Dyfnwal ap Edynfed
+}
+450.1.1={
+	holder = 159822 # Ynyr ap Dyfnwal
+}
+500.1.1={
+	holder = 159823 #Caradog Freichfras
+}
+530.1.1={
+	holder = 159824 #Meurig
+}
+550.1.1={
+	holder = 159825
+}
+560.1.1={
+	holder = 159830
+}
+590.1.1={
+	holder = 159831
+}
+600.1.1={
+	holder = 159834
+}
+605.1.1={
+	holder = 159832
+}
+645.1.1={
+	holder = 159833
+}
+665.1.1={
+	holder = 159835
+}
+680.1.1={
+	holder = 159116
+}
+710.1.1={
+	holder = 159115
+}
+740.1.1={
+	holder = 159114
+}
+780.1.1={
+	holder = 159113
+}
+790.1.1={
+	holder = 159112
+}
+810.1.1={
+	holder = 159111
+}
+830.1.1={
+	holder = 159110
+}
+840.1.1={
+	holder = 159109
+}
+909.1.1={
+	holder = 81823 #Hywel Dda
+}
+950.1.1={
+	holder = 81822 #Owain ap Hywel
+}
+987.1.1={
+	holder = 81821 #Maredudd ap Owain
+}
+999.1.2={
+	holder = 159001 # Cynan ap Hywel, Idwal Foel (the bald)'s great-grandson
+}
+1005.1.1={
+	holder = 138490 #Cadell ap Einion, joint reign
+}
+1011.1.1={
+	holder = 138494 #Owain ap Einion, joint reign
+}
+1018.1.1={
+	holder = 40500 #Llywelyn ap Seisyll
+}
+1023.1.1={
+	holder = 81863 #Rhydderch ap Iestyn, father Iestyn ap Owain ap Morgan Hen (son of #40510, Owain Morgan) is missing
+}
+1033.1.1={
+	holder = 40512 #Placeholder for correct naming sequence, Hywel ab Edwin (correct ruler of Deheubarth) and Maredudd ab Edwin, sons of Edwin ab Einion (#138492), are missing
+}
+1045.1.1={
+	holder = 81862 #Gruffydd ap Rhydderch
+}
+1055.1.1={
+	holder = 40502 #Gruffydd ap Llywelyn
+}
+1066.1.1={
+	holder=138496 #Maredudd ap Owain ap Edwin
+	#law = agnatic_succession
+	#law = succ_gavelkind
+}
+1070.1.1={
+	holder = 81815 #Rhys ap Owain ap Edwin
+}
+1076.1.1={
+	holder = 81816 #Rhys ap Tewdwr
+}
+1093.4.1={
+	holder = 152
+}
+1094.1.1={
+	holder = 41032
+}
+1098.1.1={
+	holder = 41004
+}
+1102.1.1={
+	holder = 202992
+} 
+1135.1.1={
+	holder = 214810 #Gruffyd ap Rhys
+}
+1137.1.1={
+	holder = 138498 #Anarawd ap Gruffyd
+}
+1143.1.1={
+	holder = 138500 #Cadell ap Gruffyd
+}
+1153.1.1={
+	holder = 81814 #Maredudd ap Gruffyd
+}
+1157.1.1={
+	holder = 214800 #Rhys ap Gruffyd
+}
+1197.4.28={
+	holder = 204510 
+}
+1199.4.6={
+	holder = 204514
+}
+1216.10.28={
+	holder = 454522
+}
+1219.5.4={
+	liege = k_england
+	holder = 7348 #William Marshal
+}
+1272.11.16={
+	holder = 454520
+}
+1307.7.8={
+	holder = 454510 #Edward II of England
```

Comment:
WtWSMS doesn't have this character, presumably some older ones too.

Reply:
It was a copy of the one of a county, but in the meantime the one of the county had changed. Fixed now.